src/components/List/VirtualizedListItems/VirtualizedListItems.jsx

```diff
@@ -0,0 +1,19 @@
+import cx from "classnames";
+import { VirtualizedList } from "components";
+import {
+  useChildrenAsDataObjects,
+  useListItemsRenderer
+} from "components/List/VirtualizedListItems/VirtualizedListItemsHooks";
```

Comment:
I'm not sure why this split is needed - why not having all the hooks in this file? I think it is more convenient

Reply:
I prefer to keep separating the hooks to different file from the component itself.
Maybe the hooks itself are not needed? I do prefer to move big stand alone logic outside to hooks when possible so I prefer to keep it like this, but I can remove the hooks if you think it is better.

Reply:
I don't see the login seperating the logic here, since it is not for using it somewhere else. It is very coupled to this component, and it is more convenient to have the code in one place if possible. It also not that long